scripts/gen_ssl_cert.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python2
+#
+# Script creates self signed SSL cert and the key necessary to sign it.
+# This is based off of the code in this article:
+#
+#          the common name with the servers FQDN.
+import argparse
+from OpenSSL import crypto
+from socket import gethostname
+import os
+
+
+def create_key(key_file):
+    """Create SSL key in specified key file."""
+    key = crypto.PKey()
+    key.generate_key(crypto.TYPE_RSA, 4096)
+    open(os.path.join(key_file), "wt").write(
```

Comment:
BTW, this was an excellent introduction to with: http://effbot.org/zone/python-with-statement.htm